### ISSUE TITLE
Should we move A.4.3,A.4.4 and A.4.5  out of the A.4 Dependency Descriptor Format section?

### ISSUE THREAD

**Issue by murillo128** [AOMediaCodec/av1-rtp-spec#175]
Currently we have the following structure:

```
A.1 Introduction
A.2 Conventions, Definitions and Acronyms
A.3 Media Stream Requirements
A.4 Dependency Descriptor Format
 A.4.1 Syntax
 A.4.2 Semantics
 A.4.3 Deciding Decodability using Chains
 A.4.4 Switching
 A.4.5 Templates
A.5 Signaling (Setup) Information
```
A.4.3,A.4.4 and A.4.5 doesn't seem to be correctly placed under the `A.4 Dependency Descriptor Format` and I think that they deserve to be placed on a top level index. 

Also to improve readability, I would prefer to place them closer to the introduction section, where the concepts of DD, chains are frame references are explained.

Something like:

```
A.1 Introduction
A.2 Conventions, Definitions and Acronyms
A.3 Media Stream Requirements 
A.4 Deciding Decodability using Chains
A.5 Switching
A.6 Templates
A.7 Dependency Descriptor Format
 A.7.1 Syntax
 A.7.2 Semantics
A.8 Signaling (Setup) Information
```

Maybe A.3 and A.4 can be merged (A.3 is really small) and kind of overlaps with A.4.

**Comment by DanilChapovalov** [comment]
The new structure looks better to me.
Though I would leave Templates section inside the Format (While Chain and Switching sections are about how to use DD information, Templates concept and section is about how to transfer the DD information over the wire)

**Comment by murillo128** [comment]
I think it is important to explain templates before entering the syntax/semantics, how about this?
```
A.1 Introduction
A.2 Conventions, Definitions and Acronyms
A.3 Media Stream Requirements 
A.4 Deciding Decodability using Chains
A.5 Switching
A.6 Dependency Descriptor Format
 A.6.1 Templates
 A.6.1 Syntax
 A.6.2 Semantics
A.7 Signaling (Setup) Information
```

**Comment by DanilChapovalov** [comment]
That structure looks good to me (and purely better than current structure)

(I've also thought a bit about merging A.3 "Media Stream Requirements" with A.4 "Deciding Decodability using Chains".
"Media Stream Requirements" might also be seen as "Deciding Decodability using Frame References".
I think it is important to keep them separately but may be expand A.3 as part of issue#174)

**Comment by StephenBotzko** [comment]
Addressed in PR #179
